README.md

```diff
@@ -47,6 +47,10 @@ https://availity-react-preview.netlify.com
 $ npm run new
 ```
 
+## Additional Notes
+We use `data-testid` for for unit tests to use for grabbing unique attributes. If you are using `availity-workflow@6.0.0` or later it will be removed for you in production builds. However, if you are not you will need to add the following babel plugin to remove it: 
```

Comment:
"for for" s/b "for our" ?

I had trouble understanding that sentence. I think this  reads better as:
> We use `data-testid` in our unit tests to grab unique attributes...